hibernate-core/src/main/java/org/hibernate/tool/schema/internal/Helper.java

```diff
@@ -55,22 +54,38 @@ public static ScriptSourceInput interpretScriptSourceSetting(
 			final String scriptSourceSettingString = scriptSourceSetting.toString();
 			log.debugf( "Attempting to resolve script source setting : %s", scriptSourceSettingString );
 
-			// setting could be either:
-			//		1) string URL representation (i.e., "file://...")
-			//		2) relative file path (resource lookup)
-			//		3) absolute file path
-
-			log.trace( "Trying as URL..." );
-			// ClassLoaderService.locateResource() first tries the given resource name as url form...
-			final URL url = classLoaderService.locateResource( scriptSourceSettingString );
-			if ( url != null ) {
-				return new ScriptSourceInputFromUrl( url, charsetName );
+			final String[] paths = scriptSourceSettingString.split( "\\s*,\\s*" );
```

Comment:
Using a compiled Pattern might be much more performant, unless the parameter of String#split() is one char long (JVM have optimization to use the char directly without incurring reg compiling).